src/eigen/lobpcg_hyper_impl.jl

```diff
@@ -100,6 +100,28 @@ end
 
 Base.:*(Aadj::Adjoint{T,<:LazyHcat}, B::AbstractMatrix) where {T} = Aadj * LazyHcat(B)
 
+# Special case of Hermitian result: can only actively compute the block upper diagonal
```

Comment:
Describe what the function does instead of how it's implemented: something like "Computes A*B, assuming the result is hermitian"